gradle/project.gradle.kts

```diff
@@ -49,6 +50,10 @@ val javaSupportedVersions = arrayOf(JavaVersion.VERSION_1_9, JavaVersion.VERSION
     }
 }
 
+javafx {
+    modules = listOf("javafx.controls", "javafx.fxml", "javafx.media", "javafx.swing", "javafx.web")
```

Comment:
All modules should be added. This isnt all is it?

Reply:
https://github.com/openjfx/javafx-gradle-plugin/blob/master/src/main/java/org/openjfx/gradle/JavaFXModule.java

Reply:
Im thinking whether using enum constants is better idea and whether using JavaFXModule.values() is not even better.